src/rock/models/binary-files/model.js

```diff
@@ -55,10 +57,22 @@ export class BinaryFile extends Rock {
     })
       .then(this.getFromIds.bind(this))
     );
-
   }
 
-
+  async attachPhotoIdToUser({ personId, previousPhotoId, newPhotoId } = {}) {
+    try {
+      await api.patch(`/People/${personId}`, {
+        PhotoId: newPhotoId,
+      });
+      if (!isEmpty(previousPhotoId)) {
+        try {
+          await api.delete(`/BinaryFiles/${previousPhotoId}`);
+        } catch (e) {} // eslint-disable-line
```

Comment:
I'm speaking out of ignorance here, but can this `.delete()` method return an error that show be handled?

Reply:
@IsaacHardy I agree with you but I'm not sure how we would handle it. This is coming (partially) from [here](https://github.com/NewSpring/Holtzman/blob/master/imports/deprecated/methods/files/server/avatar.js#L31)

Reply:
Ah, I see. I'm going to ask Rich to see if he knows how we might be able to handle that potential catch.